packages/web/src/components/data-entry/FormTile.js

```diff
@@ -594,23 +479,11 @@ const AdvancedForm = (props) => {
         <GatedContentUploadPromptModal
           onSubmit={() => {
             props.toggleAdvanced()
-            setIsAvailabilityModalOpen(true)
+            // TODO: re-enable this before merge
+            // setIsAvailabilityModalOpen(true)
```

Comment:
I don't know what this is for. @sddioulde ?

Reply:
good q, don't know the answer

Reply:
Per discussion in slack, will remove this popup feature entirely.

Reply:
Let's make a separate ticket and not block on this - we can have someone on money team take it too, not high pri

Reply:
https://linear.app/audius/issue/PAY-1733/remove-gatedcontentuploadprompt